requirements-dev.txt

```diff
@@ -2,3 +2,4 @@ fastjsonschema
 jsonschema
 jsonschema-rs
 ddt
+qiskit-ibm-runtime@git+https://git@github.com/Qiskit/qiskit-ibm-runtime.git
```

Comment:
Originally it was `qiskit-ibm-runtime>=0.22` but I changed it because we had test failures due to obsolete code. I'll try switching back, now that 0.2.3 is out, and see if it works.